dev-clean/headers/llsc/cli.hpp

```diff
@@ -1,6 +1,11 @@
 #ifndef LLSC_CLI_HEADERS
 #define LLSC_CLI_HEADERS
 
+#include "filesys.hpp" // for initial_fs, default_sym_file_size
+#include "../llsc.hpp" // for DEBUG
```

Comment:
again, `llsc.hpp` already includes `cli.hpp`, there is no need to include `llsc.hpp` in `cli.hpp`.